be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -49,13 +50,15 @@ Status PipelineDriver::prepare(RuntimeState* runtime_state) {
         all_local_rf_set.insert(rf_set.begin(), rf_set.end());
 
         const auto* global_rf_collector = op->runtime_bloom_filters();
+        auto is_scan_operator = dynamic_cast<OlapScanOperator*>(op.get()) != nullptr;
```

Comment:
Why stick to `OlapScanOperator`?  Could you apply this mechanism to all scan operators?

Reply:
this mechanism is experimental, OlapScanOperator wait the same time as other Operators, in the future, maybe deepmost ScanOperator should wait for more time. now ignore this mechanism.

Reply:
Please take other scan operator into account. Otherwise it's other people's burden to implement this feature.